nerfstudio/scripts/render.py

```diff
@@ -1159,16 +1159,12 @@ def update_config(config: TrainerConfig) -> TrainerConfig:
                             if "ray_drop_prob" in lidar_output:
                                 points_in_local = points_in_local[(lidar_output["ray_drop_prob"] < 0.5).squeeze(-1)]
 
-                            points_in_world = (
-                                lidar.lidar_to_worlds[0]
-                                @ torch.cat(
-                                    [
-                                        points_in_local,
-                                        torch.ones_like(points_in_local[..., :1]),
-                                    ],
-                                    dim=-1,
-                                ).unsqueeze(-1)
-                            ).squeeze()
+                            points_in_world = to_world(lidar.lidar_to_worlds[0], points_in_local)
```

Comment:
you can use `transform_points` in `lidars.py` instead of the `to_world`

Reply:
Sure, we've simply put the code used in previous version of render to a function, did not checked that a function actually existed in lidar.py. We'll do the changes.

Reply:
Done